### PR TITLE
Able to write stats data in multiple OpenTSDB dabases

### PR DESCRIPTION
**Pull request by surabujin**
Create separate "channel" of stats datapoint processing for each defined
into config file OpenTSDB target/host.
    
Each "channel" have own kafka spout (with own group-id), so in case of
errors and "tuple repeats" done by storm everything will work correct.

Part of #4925

### CODE REVIEW

File: confd/vars/docker-compose.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
-#no_gui: true
+no_gui: true
```
[the remark this answers is not in the document]
Comment (surabujin) [reply confd/vars/docker-compose.yaml:2]: Yes.

**Review by niksv — APPROVED** [review]
Pr has wrong description (with new internal kafka topic)

**Comment by pablomuri** [comment]
The last change just rebases the branch with develop.

File: src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java [hunk 2]
```diff
@@ -65,59 +73,113 @@ public StormTopology createTopology() {
...
+        for (Map.Entry<Object, Object> entry : rawTopologyConfig.entrySet()) {
+            String property = (String) entry.getKey();
+            if (! property.startsWith(prefix)) {
+                continue;
+            }
+
+            String name = property.substring(prefix.length());
+            if (name.isEmpty()) {
+                throw new IllegalArgumentException(String.format(
+                        "Target name is empty string (property: \"%s\", prefix: \"%s\")", property, prefix));
```
Comment (dmitrii-beliakov) [inline src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java:165]: The target's name is an empty string

File: src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java [hunk 3]
```diff
@@ -65,59 +73,113 @@ public StormTopology createTopology() {
                 getZookeeperConfig());
         declareSpout(tb, zooKeeperSpout, ZooKeeperSpout.SPOUT_ID);
 
+        if (targets.isEmpty()) {
+            throw new IllegalStateException("There is no any OpenTSDB endpoint(target) defined into topology config");
+        }
+
+        int expectedEnableDisableBoltsCount = 0;
+        List<String> enableDisableBolts = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (pablomuri) [reply src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java:81]: Actually, there are not disabled. I've change the name

File: src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java (hunk 2 above)
Comment (pablomuri) [reply src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java:165]: done

File: src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java [hunk 4]
```diff
@@ -65,59 +73,113 @@ public StormTopology createTopology() {
...
+            }
+
+            String name = property.substring(prefix.length());
+            if (name.isEmpty()) {
+                throw new IllegalArgumentException(String.format(
+                        "Target name is empty string (property: \"%s\", prefix: \"%s\")", property, prefix));
+            }
+
+            String value = (String) entry.getValue();
+            if (value == null || value.isEmpty()) {
```
[the remark this answers is not in the document]
Comment (pablomuri) [reply src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java:169]: done

File: src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java [hunk 5]
```diff
@@ -65,59 +73,113 @@ public StormTopology createTopology() {
...
+                continue;
+            }
+
+            targets.put(name, (String) entry.getValue());
```
[the remark this answers is not in the document]
Comment (pablomuri) [reply src-java/opentsdb-topology/opentsdb-storm-topology/src/main/java/org/openkilda/wfm/topology/opentsdb/OpenTsdbTopology.java:173]: done

**Review by dmitrii-beliakov — APPROVED** [review]

File: confd/vars/main.yaml [hunk 6]
```diff
@@ -20,6 +20,8 @@ kilda_zookeeper_reconnect_delay_ms_server42: 10000
 kilda_zookeeper_reconnect_delay_ms_speaker: 1000
 kilda_opentsdb_hosts: "opentsdb.pendev"
 kilda_opentsdb_port: "4242"
+kilda_victoriametrics_host: "victoriametrics.pendev"
+kilda_victoriametrics_port: "4242"
```
Comment (niksv) [inline confd/vars/main.yaml:24]: Opentdstb and victoria have equal port 4242. As we can run both DBs for local env will we have problems with "Address 127.0.0.1:4242 is already in use"?

**Review by dmitrii-beliakov — APPROVED** [review]

**Review by dmitrii-beliakov — APPROVED** [review]

**Review by pablomuri — CHANGES_REQUESTED** [review]
Add an explanation about the differences between the victoria metrics and opentsdb API to read

**Review by pablomuri — APPROVED** [review]

**Pull request merged**